lib/node/Creator.js

```diff
@@ -71,6 +71,13 @@ const NodeCreator = {
   nthRoot (radicandNode, rootNode) {
     const symbol = NodeCreator.symbol('nthRoot');
     return new math.expression.node.FunctionNode(symbol, [radicandNode, rootNode]);
+  },
+  powerTerm (base, exponent) {
```

Comment:
this is better than the constantTerm, but I still think we can get away with not having be a node type at all. maybe that'll get too complicated though. feel free to message me on gitter and we can chat back and forth more if you want!

https://gitter.im/mathsteps-chat/Lobby